GVFS/GVFS.Service/GVFSMountProcess.Mac.cs

```diff
@@ -1,52 +1,63 @@
 ﻿using GVFS.Common;
-using GVFS.Common.Tracing;
-using GVFS.Service.Handlers;
-using System;
-
+using GVFS.Common.Tracing;
+using System.Diagnostics;
+
 namespace GVFS.Service
 {
-    public class GVFSMountProcess : IDisposable
+    public class GVFSMountProcess : IRepoMounter
     {
-        private const string ParamPrefix = "--";
+        private const string ExecutablePath = "/bin/launchctl";
+        private const string GVFSPath = "/usr/local/vfsforgit/gvfs";
 
-        private readonly ITracer tracer;
+        private MountLauncher processLauncher;
+        private bool waitTillMounted;
 
-        public GVFSMountProcess(ITracer tracer, int sessionId)
+        public GVFSMountProcess(MountLauncher processLauncher = null, bool waitTillMounted = false)

```

Comment:
I didn't see where this was called with `waitTillMounted` `true`.  Did I miss something?  Also seems like this is only for the unit tests.  Can we find another way instead of adding something that is only for the tests?

Reply:
Removed it.